srsenb/src/stack/rrc/rrc.cc

```diff
@@ -354,6 +359,75 @@ bool rrc::setup_ue_ctxt(uint16_t rnti, LIBLTE_S1AP_MESSAGE_INITIALCONTEXTSETUPRE
   return true;
 }
 
+bool rrc::modify_ue_ctxt(uint16_t rnti, LIBLTE_S1AP_MESSAGE_UECONTEXTMODIFICATIONREQUEST_STRUCT *msg)
+{
+  bool err = false;
+  pthread_mutex_lock(&user_mutex);
+
+  rrc_log->info("Modifying context for 0x%x\n", rnti);
+
+  if(users.count(rnti) == 0) {
```

Comment:
here and in a few other lines the clang-format is not applied.

Reply:
Do you have some kind of guide on how to develop for srsLTE?  I have never heard of "clang-format" before. If there is some kind of automatically enforced coding style, do you have something like pre-commit hooks that one can use so git will automatically perform those?  Or maybe configuration for vim to apply the desired formatting?

Reply:
ok, I guess I have figured out at least "a" way to use clang-format.  Interestingly, when I wrote the related code (without space between if and the parenthesis) I had to force myself to do it, as I saw the surrounding code was using that.  In Osmocom we use Linux Kernel coding style, and as "if" is not a function, there clearly is a space after it ;)

Reply:
we've introduced a clang-format style file about a year ago I believe and all new code written since then uses this format. However, there is still some "old" code that has, on purpose, not been converted in, e.g., one large clang-format commit. So we will have mixed style until we touch every line at least once more ;-)
We internally all use [this](https://github.com/barisione/clang-format-hooks) commit hook to enforce the formatting. We are working on a script to check this for the public pull requests as well but until this is there we have to manually check this.